### ISSUE TITLE
[BUG] Super event: number of sub-event limitation via the app?

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#3790]
The IlliniFest super event on Oct 14, 2023 has over 100 sub-events as published by the old Event Manager. The sub-events over 100 are not displaying in the app. 

Bing mentioned that there is no sub-event limit set in the events bb. Is there a sub-event limit set on the app or calendar bb side? If yes, can it be increased to 150? 

v5.0.74

**Comment by vburgett** [comment]
Or is 100 events perhaps a limitation set by Groups since Event Manager permissions (and event limits?) are tied to Groups. This event is connected to the below Group. 

![image](https://github.com/rokwire/illinois-app/assets/1044905/a16f7e7b-c62b-435d-af38-50addffbc380)

![image](https://github.com/rokwire/illinois-app/assets/1044905/1ed67dcd-6cd6-4673-b9db-351e4114aa35)

**Comment by dobromirdobrev** [comment]
Hi @vburgett ,

This is fixed in app version 5.0.76. Please verify when you get a chance and let us know. Thanks.

**Comment by vburgett** [comment]
In 5.0.76, the app is only displaying IlliniFest sub-events up to 100 events still. I believe I will need to wait to test this until after the 4am CT event building block sync on Saturday, October 7.

**Comment by vburgett** [comment]
Confirming this is fixed in version 5.0.76.

**Comment by vburgett** [comment]
In 5.0.77, I am still seeing this super-event cutting off sub-events at 100 events. The last event displayed is State Farm Center Basketball Court Photo from 2-4pm when there should be these 6 published sub-events following it: 

Art & Design Facility Tour (Afternoon)  
2:30pm-3:15pm
ID650dff7f4c147c00078fe14d

Liberal Arts & Sciences (LAS) Mini-Lecture: Severe & Hazardous Weather (Afternoon)
3-3:30pm
65032020586721000a0de6f2

Liberal Arts & Sciences (LAS) Mini-Lecture: Why Studying the Media Matters to Latinas/os & Latinxs (Late Afternoon)
2:30-3pm
6503202030ca5a00086da127

Prospective Transfer Presentation: Office of Undergraduate Admissions (Afternoon)
2:30-3pm
6503202116fa9f000a5073e7

College of Education Information Presentation (Late Afternoon)
2:30-3pm
6503202358672100092678e7

Krannert Center for the Performing Arts Tour (Late Afternoon)
3-3:30pm
6503201e40a0ac0007208551

In the Group (Enrollment Management Event Admins) that is setting the permissions for this event , it reads as "188" events" when there are 106 sub-events. I see various sub-events duplicated. Is this part of the issue?

Any thoughts on what is causing this issue? @dobromirdobrev @pmarkhennessy @minump @bingzhang @sandeep-ps 
The event is October 14, and they have 12 more events they would like to add asap.

**Comment by bingzhang** [comment]
@vburgett I check this super event in events BB in prod. it has 106 subevents.

**Comment by dobromirdobrev** [comment]
Hi @vburgett ,

I presume that the super event that we are talking about here is `IlliniFest: Campus-Wide Open House for Prospective Students` with id `6508e08e16fa9f000a507424`.

After some more investigation, we identified 3 different problems with the sub events:

1. The first one is that the app was limiting number of events in the group events panel to 100 (ref: https://user-images.githubusercontent.com/1044905/273044707-1ed67dcd-6cd6-4673-b9db-351e4114aa35.png ).
This was the problem which was already fixed in version 5.0.76

2. The second one is that the app is limiting the number of events in the events detail panel to 100. This was fixed today and is available in version 5.0.78.

3. The third one is that the app is showing sub events in the group events panel which should not be shown. (e.g. it shows 188 events of which 106 are sub-events). This is a problem on the backend and a relevant issue was created https://github.com/rokwire/groups-building-block/issues/397 which needs to be fixed

As a conclusion two of the problems are fixed in version 5.0.78 and there is one outstanding which is a backend issue.

cc @pmarkhennessy
